<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="8" rowspan="4" colspan="2">
     <widget class="QLabel" name="label_pic">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="10" rowspan="4" colspan="2">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>231</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="13" rowspan="11">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Stencil</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GO!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Stencil</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>YOU</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Stencil</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CAN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Stencil</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HOT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Stencil</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HOW</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="3" rowspan="2" colspan="2">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>140</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="12" rowspan="10">
     <widget class="QSlider" name="verticalSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
      <property name="tickInterval">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="8" column="6" rowspan="4" colspan="2">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>200</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="8">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>666</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" rowspan="2" colspan="8">
     <widget class="QLabel" name="nameplate">
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>24</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>EL FUEGO  THE HOUSE OF SPICE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="8" rowspan="4" colspan="4">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>34</width>
        <height>88</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="10">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>416</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" rowspan="7">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>341</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="5" rowspan="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>364</width>
        <height>158</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="7" rowspan="3">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>165</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0" colspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="grams">
        <property name="font">
         <font>
          <family>Stencil</family>
         </font>
        </property>
        <property name="text">
         <string>Grams</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ounces">
        <property name="font">
         <font>
          <family>Stencil</family>
         </font>
        </property>
        <property name="text">
         <string>Ounces</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="11" column="0" colspan="6">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QRadioButton" name="NoOfPpl_Two">
        <property name="font">
         <font>
          <family>Stencil</family>
         </font>
        </property>
        <property name="text">
         <string>2 Serving</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="NoOfPpl_Three">
        <property name="font">
         <font>
          <family>Stencil</family>
         </font>
        </property>
        <property name="text">
         <string>3 Serving</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="NoOfPpl_Four">
        <property name="font">
         <font>
          <family>Stencil</family>
         </font>
        </property>
        <property name="text">
         <string>4 Serving</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="NoOfPpl_One">
        <property name="font">
         <font>
          <family>Stencil</family>
         </font>
        </property>
        <property name="text">
         <string>1 Serving</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="9" rowspan="3" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>12</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="5" rowspan="3" colspan="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>431</width>
        <height>92</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>74</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="recipeLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_nameDescription">
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>label_name</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" rowspan="3">
     <widget class="QWidget" name="checkBoxesWidget" native="true"/>
    </item>
    <item row="3" column="2" rowspan="3">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>92</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHome"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
